Delver/Assets/Source/Gameplay/Abilities/AttackCollider.cs

```diff
@@ -21,25 +21,32 @@ public class AttackCollider : MonoBehaviour
     protected Collider2D[] overlapsThisFrame;
 
     public Vector3 GetBounds() { return boxDefinition.size; }
-    
 
-    protected void OnEnable()
+    public float speed = 0;
```

Comment:
Depending on if you want this visible in the editor, Id mark it [HideInInspector] or [SerializeField] just to make it obvious